tests/fixtures/response/app/routes/return-status-code-418.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  fastboot: Ember.inject.service(),
+
+  model() {
+    return {
+      statusCode: 418
+    };
+  },
+
+  afterModel(model) {
+    this.set('fastboot.response.statusCode', model.statusCode);
```

Comment:
If you are running late don't worry. I won't be getting out before 4:30 if not a few minutes later. Lmk when your in the area